speech/unit_tests/test_alternative.py

```diff
@@ -15,41 +15,41 @@
 import unittest
 
 
-class TestTranscript(unittest.TestCase):
+class AlternativeTests(unittest.TestCase):
```

Comment:
s/`AlternativeTests`/`TestAlternative` (this is a general rule of them, name a `TestCase` subclass something like `TestFoo`)